Makefile

```diff
@@ -130,6 +130,10 @@ docker-build-ci:
 docker-push:
 	docker push ${IMG}
 
+release:
+	@sed -i -e 's/^VERSION := .*/VERSION := ${NEWVERSION}/' ./Makefile
+	@sed -i'' -e 's@image: .*@image: '"$(REPOSITORY):$(NEWVERSION)"'@' ./config/manager/manager.yaml ./deploy/gatekeeper.yaml
```

Comment:
We may want to match against repository and old version here. That way if we add other pods, we wont mistakenly clobber their image paths